py/dml/compat.py

```diff
@@ -99,9 +99,21 @@ class function_in_extern_struct(CompatFeature):
     } my_interface_t;
     ```
     '''
-    short = 'Disallow non-pointer function members in extern structs'
+    short = 'Allow non-pointer function members in extern structs'
     last_api_version = api_7
 
+
+@feature
+class optional_version_statement(CompatFeature):
+    '''When this compatibility feature is enabled, the `dml 1.4;`
```

Comment:
> The version specification statement (e.g. `dml 1.4;`) at the start of each file is optional

Reply:
I would agree that this clarification would be needed if this were in a non-dying part of the spec, but this is a dying path so easy-to-read is more important than formal correctness. It's easy to understand that it refers to `dml 1.2;` in a 1.2 context.